backend/lib/rpc-service/data-types.js

```diff
@@ -4,11 +4,11 @@ const ClaimEvent = Type.Object({
   client: Type.Number(),
   provider: Type.Number(),
   pieceCid: Type.Any(),
```

Comment:
Is there a way how to use `CID` from multiformats here? For example, can we teach Typebox about `CID` as a custom type?